### ISSUE TITLE
I think Finn just open sourced their Layout framework

### ISSUE THREAD

**Issue by eonist** [Torsph/DynamicLayout#1]
https://github.com/schibsted/layout

Today actually. So readme is void ;)

**Comment by Torsph** [comment]
Yeah, but it is actually something different :) Its a replacement for xib/nib rather then a template language :)

**Comment by eonist** [comment]
Ah, cool. Hard to interpret what it was. Seems cool tho. I think defining app structure via json is a good approach right now.
